website/src/components/Feature.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import styles from '../css/index.module.css';
```

Comment:
You can use `@site/src/css/index.module.css` instead of relative paths so that if you move this component around it wouldn't break :)

Reply:
It's only one level up, so I don't mind, but thanks for suggestion!